package.json

```diff
@@ -66,6 +66,7 @@
     "generate-search-parser": "peggy --format es -o src/libs/SearchParser/searchParser.js src/libs/SearchParser/searchParser.peggy ",
     "web:prod": "http-server ./dist --cors"
   },
+
```

Comment:
```suggestion
```
NAB

Reply:
Done! and sorry for such a silly mistake